admin-dev/themes/new-theme/js/pages/order/create.js

```diff
@@ -61,10 +61,22 @@ function refreshCart() {
   orderPageManager.refreshCart();
 }
 
+/**
+ * proxy to allow other scripts within the Create Order page to refresh cart
+ */
+function addCreatedCartRuleToCart(cartRuleId) {
+  if (orderPageManager === null) {
+    console.log('Error: Could not refresh created cart rule to the current cart as orderPageManager is null');
```

Comment:
I think this sentence can be improved 😅  as I dont fully understand it.

```suggestion
    console.log('Error: Cannot apply created cart rule to the current cart as orderPageManager is not available');
```